### ISSUE TITLE
Exit code 0 when runtime error happened in loaded javascript configuration

### ISSUE THREAD

**Issue by nkabardin** [stylelint/stylelint#2115]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

When you have a `stylelint.config.js` looking like this:

```javascript
module.exports = require("./someUnknownStuff.js");
```

and you don't have `someUnknownStuff.js`, then stylelint cli breaks, obviously, and shows and error. But returning 0 (success) exit code, so build on jenkins is succeeded, even though configuration is wrong.

> Which rule, if any, is this issue related to?

Not related to any rule

> What CSS is needed to reproduce this issue?

No CSS involved

> What stylelint configuration is needed to reproduce this issue?

```javascript
//styleling.config.js
module.exports = require("./someUnknownStuff.js");
```
> Which version of stylelint are you using?

`7.6.0`, and verified problem on current `master` as well

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No

> What did you expect to happen?

Error code 1 or other that != 0;

> What actually happened (e.g. what warnings or errors you are getting)?

Error code 0

**Comment by nkabardin** [comment]
PR with fix: https://github.com/stylelint/stylelint/pull/2116

I also tried to create a system test for that, but seems like system tests are not covering CLI at the moment, and I can help with that and add system tests for CLI, but I would like to ask for advice on what's the best way to do that.

**Comment by nkabardin** [comment]
new PR submitted: https://github.com/stylelint/stylelint/pull/2140

system test created!
